### ISSUE TITLE
"event" lost in primeng package 

### ISSUE THREAD

**Issue by huineng** [primefaces/primeng#8104]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
```

For some reason i do not know the primeng code in npm is loosing event parameters

This generates bugs in older browsers for my application "event is undefined"

 "primeng": "^8.0.2",
angular 8 

let me show this with an example

code in prime
code in npm

example from dropdown.js https://github.com/primefaces/primeng/blob/master/src/app/components/dropdown/dropdown.ts#L399

```js
onItemClick(event, index) {
        const option=event.option;
        this.itemClick = true;

        if (this.virtualScroll) {
            this.virtualScrollSelectedIndex = index;
        }
        if (!option.disabled) {
            this.selectItem(event, option);
            this.focusViewChild.nativeElement.focus();
        }

        setTimeout(() => {
            this.hide(event);
        }, 150);
    }
```

final published code in npm (and installed on our machine

https://unpkg.com/primeng@8.0.2/components/dropdown/dropdown.js

```
Dropdown.prototype.onItemClick = function (event, index) {
        var _this = this;
        var option = event.option;
        this.itemClick = true;
        if (this.virtualScroll) {
            this.virtualScrollSelectedIndex = index;
        }
        if (!option.disabled) {
            this.selectItem(event, option);
            this.focusViewChild.nativeElement.focus();
            this.filled = true;
        }
        setTimeout(function () {
            _this.hide();
        }, 150);
    };
```

the event is gone on this.hide() and as such causing  a problem

```js
Dropdown.prototype.hide = function () {
        this.overlayVisible = false;
        if (this.filter && this.resetFilterOnHide) {
            this.resetFilter();
        }
        if (this.virtualScroll) {
            this.virtualAutoScrolled = false;
        }
        this.cd.markForCheck();
        this.onHide.emit(event);
    };
```

the onhide emits now an undefined event and is no longer working

**Comment by huineng** [comment]
following line items are also missing an event as prop

```
bindDocumentClickListener() {
        if (!this.documentClickListener) {
            this.documentClickListener = this.renderer.listen('document', 'click', (event) => {
                if (!this.selfClick && !this.itemClick) {
                    this.hide(event);
                    this.unbindDocumentClickListener();
                }
                
                this.clearClickState();
                this.cd.markForCheck();
            });
        }
    }
```

```
onWindowResize() {
        if (!DomHandler.isAndroid()) {
            this.hide(event);
        }
    }
```

it remains a complete mystery to me why if i build the components using './node_modules/.bin/ngc -p tsconfig-release.json' the events are *not* lost, but on npm they are not there

**Comment by mertsincan** [comment]
Fixed for 8.0.3.

Best Regards,
